lib/kazoo/subscription.rb

```diff
@@ -0,0 +1,182 @@
+module Kazoo
+  class Subscription
+    attr_reader :timestamp, :version
+
+    def self.everything
+      create(/.*/)
+    end
+
+    def self.create(subscription, pattern: :white_list, timestamp: Time.now)
```

Comment:
`subscription_type` might be a better name than pattern, which kinda implies regex